src/store/synchronizer.js

```diff
@@ -0,0 +1,30 @@
+let subscribedStores = [];
```

Comment:
Not tryna be captain obvious but basically the client store here is diff to the server store and every action is logged? I'm still unsure about line 9 in this file

Reply:
no idea what you're talking about m80